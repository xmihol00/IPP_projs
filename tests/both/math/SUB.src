.IPPcode21

DEFVAR GF@num
DEFVAR GF@int
MOVE GF@int int@25

SUB GF@num int@0 int@15
WRITE GF@num
WRITE string@\010

SUB GF@num GF@int int@12
WRITE GF@num
WRITE string@\010

SUB GF@num  GF@int int@-35
WRITE GF@num
WRITE string@\010

SUB GF@num GF@num GF@int
WRITE GF@num
WRITE string@\010

SUB GF@num GF@num GF@num
WRITE GF@num
WRITE string@\010

WRITE string@\010

CREATEFRAME
DEFVAR TF@_
DEFVAR TF@!
MOVE TF@_ int@12

SUB TF@! TF@_ TF@_
WRITE TF@!
WRITE string@\010

SUB TF@! TF@! int@-18
WRITE TF@!
WRITE string@\010

SUB TF@_ int@2 TF@!
WRITE TF@_
WRITE string@\010

SUB TF@_ int@1000 int@999
WRITE TF@_
WRITE string@\010

WRITE string@\010

PUSHFRAME
WRITE LF@_
WRITE string@\010
WRITE LF@!
WRITE string@\010

SUB LF@_ LF@_ int@-999
WRITE LF@_
WRITE string@\010

SUB LF@! int@4 LF@!
WRITE LF@!
WRITE string@\010

SUB LF@! LF@_ LF@!
WRITE LF@!
WRITE string@\010

SUB LF@_ int@5 int@-5
WRITE LF@_
WRITE string@\010
WRITE string@\010

POPFRAME
WRITE TF@_
WRITE string@\010
WRITE TF@!
WRITE string@\010
