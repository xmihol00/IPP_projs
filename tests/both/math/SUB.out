-15
13
60
35
0

0
18
-16
1

1
18
1000
-14
1014
10

10
1014
